WIP: Modern guidelines

alix-tz opened the pull request:
J'ai reformulé certaines phrases qui me paraissaient trop incorrectes en anglais et j'ajoute la prise en compte de la question des guidelines moderne. Ca veut dire:
- préciser quand une règle ne s'applique qu'aux documents médiévaux
- ajouter les règles qui s'appliquent aux documents modernes
- reformuler quand c'est nécessaire pour ne pas laisser penser qu'une règle ne s'applique qu'aux documents médiévaux.

L'état des règles pour les documents moderne est ici: https://docs.google.com/spreadsheets/d/1bFE-rRk6ZwgIHqXAOgwPo1s1zwQ-UPTLPnzjaRmTMsk/edit?gid=0#gid=0

Suivi de la progression:
- [x] general principles
- [x] letters
- [x] segmentation
- [x] abbreviations
- [x] accents, caedillas
- [x] punctuation
- [x] ligatures
- [x] corrections and other functionnal elements
- [ ] character table

alix-tz committed:
WIP: rephrasing some of the guidelines (linguistic considerations) and adding precision on the guidelines for modern documents
- [x] general principles
- |x] letters
- [ ] segmentation
- [ ] abbreviations
- [ ] accents, caedillas
- [ ] punctuation
- [ ] ligatures
- [ ] corrections and other functionnal elements
- [ ] character table

PonteIneptique commented on data/guidelines/en/index.md:
```diff
@@ -1,25 +1,19 @@
-## CATMuS, common rules for the transcription of medieval roman sources
+## CATMuS, common rules for the transcription of medieval and modern roman sources
```
From Middle Ages to Today ?

PonteIneptique commented on data/guidelines/en/numbers.md:
```diff
@@ -1,7 +1,11 @@
+
+For numbers written with superscript letters, they will be represented by superscript<!--todo: add link to section on superscripted texts--> numbers as well. 
+
+The groups of numbers should be clearly separated. The punctuation surrounding the figures should be retained where it exists. In the case of medieval documents, we recommend systematically framing them with dots, as is frequently done in manuscripts. 
```
There is a mistake here. WE SHOULD NOT add dots if there is not dots in the original manuscript.

alix-tz replied:
I'll let @matgille see this point before we correct it

PonteIneptique commented on data/guidelines/en/numbers.md:
```diff
@@ -1,7 +1,11 @@
+
+For modern documents, the case should be respected when roman numbers are used, and, when the distinction between a capital I and a 1 is difficult to make (as can be the case in typewritten documents or in some manuscript), we recommend using the transcription that makes the most sense. 
+
+<!-- should this be presented as modern-only ? -->[Old-style roman numerals](https://textcreationpartnership.org/docs/dox/rnums.html) M and D (sometimes transcribed as "CIↃ" and "IↃ") should be transcribed are, respectively, M and D.
```
I don't know. Lets see what @ArianePinche thinks

PonteIneptique commented on data/guidelines/en/ramists.md:
```diff
@@ -1,2 +1,7 @@
 For most medieval Romance languages, ramists are a distinction of form, 
```
I think having bold would be helpful in cases like that

alix-tz committed:
modifications after TC's comments

alix-tz committed:
adding the rest of the guidelines
I'm still missing a few elements but I think they can be saved for the characters part

alix-tz commented:
Now that I'm taking a look at the character table, I'm wondering if we should have an extra column for "medieval only" or "modern only" characters. What do you think?

alix-tz commented:
Note: there should be a small paragraph in "Index of characters", before the table, explaining how to read the table. For example, this would insure that "corresponding characters" is correctly understood as "rejected alternative transcriptions".

alix-tz committed:
removing comment on medieval rule for insertions.

PonteIneptique commented on data/guidelines/en/corrections_and_others.md:
```diff
@@ -5,4 +5,4 @@ of segmentation into zones and lines.
 For in-line corrections, we recommend using the combination of "⟦" [U+27E6] and "⟧" [U+27E7] to encapsulate a legible word marked as crossed out. When the word is not decipherable (because of a stain, an incomplete tracing or any other reason), "⟦" and "⟧" can be used without any letter in between ("⟦⟧"). <!-- AC: I almost want to say that this is also this symbol that should be used to mark tran ...
 
-For corrections relying on an interlinear addition, we first recommend that the interlinear addition be considered a separate text line, type accordingly using the SegmOnto vocabulary. In cases where a symbol is used in the source to mark where the interlinear addition should be inserted, we recommend to always use the same unicode character : "⎀" [U+2380] for modern and contemporary sources. <!- ...
+For corrections relying on an interlinear addition, we first recommend that the interlinear addition be considered a separate text line, type accordingly using the SegmOnto vocabulary. In cases where a symbol is used in the source to mark where the interlinear addition should be inserted, we recommend to always use the same unicode character : "⎀" [U+2380] for modern and contemporary sources but  ...
```
Is is really necessary to have two different approaches here ?

alix-tz replied:
No you're right, we could use the same symbol as catmus medieval.

We had agreed, with Simon and Ariane, that `"⎀" [U+2380]` is technically more correct than `"‸" [U+2038]` since it is an "insertion symbol". 

Now, it's easy to fix for catmus modern since it's not released yet.

alix-tz replied:
ok, pour simplifier, je bascule CATMuS Modern v1 vers `‸`

alix-tz committed:
homogeneization of the symbol for insertion
